### PR TITLE
[SD-3425] - User save fails when change is on different tabs

### PR DESCRIPTION
**Pull request by akintolga**

### CODE REVIEW

File: client/app/scripts/superdesk-users/users.js [hunk 1]
```diff
@@ -1298,22 +1313,26 @@
             };
         }])
 
-        /**
+		/**
          * @memberof superdesk.users
          * @ngdoc directive
-         * @name sdUserPrivileges
+         * @name sdUserPreferences
```
Comment (sivakuna-aap) [inline client/app/scripts/superdesk-users/users.js]: This should be privileges only not preferences.

File: client/app/scripts/superdesk-users/users.js [hunk 2]
```diff
@@ -1080,7 +1090,11 @@
                         preSaveCategoriesCheck()
                         .then(function () {
                             var update = createPatchObject();
-                            return preferencesService.update(update);
+                            return preferencesService.update(update).then(function() {
+                                userList.getUser(scope.user._id, true).then(function(u) {
+                                    scope.user = u;
+                                });
+                            });
```
Comment (plamut) [inline client/app/scripts/superdesk-users/users.js]: Would it make sense to flatten this promise chain here?

**Comment by plamut** [comment]
Looks good IMO. :+1:

**Comment by mdhaman** [comment]
:+1:

**Pull request merged**